src/expression.d

```diff
@@ -2009,7 +2008,7 @@ private:
  */
 extern (C++) int RealEquals(real_t x1, real_t x2)
 {
-    return (Port.isNan(x1) && Port.isNan(x2)) || Port.fequal(x1, x2);
+    return (TargetReal.isNaN(x1) && TargetReal.isNaN(x2)) || TargetReal.areIdentical(x1, x2);
```

Comment:
``areIdentical``?  I think it's best to stick with ``std.math`` name conventions. :-)